stages/org.osbuild.kernel-cmdline

```diff
@@ -1,13 +1,26 @@
 #!/usr/bin/python3
 import os
+import platform
 import sys
 
 import osbuild.api
 
+MAX_SIZE_MAP = {
+    "x86_64": 2048,
+    "i386": 2048,
+    "arm": 1024,
+    "aarch64": 2048,
+    "powerpc": 2048,
+    "ppc64": 2048,
+    "riscv": 1024,
+    "mips": 4096
+}
```

Comment:
I assume this doesn't change regularly but a comment of where they come from is a nice to have.

Reply:
Yes, they are pretty stable.

Sure, I can add comments for each of these. They are commonly found in `{arch}/include/asm/setup.h` or `{arch}/include/uapi/asm/setup.h`, depending on the architecture.

Reply:
Just that last sentence is enough as a comment then :)

Reply:
Oops I saw your comment after pushing. If you prefer to reduce the noise, I can revert it to just the one sentence.

Reply:
It's fine :)